tests/file_storage/tests.py

```diff
@@ -405,6 +408,48 @@ def test_delete_no_name(self):
         with self.assertRaises(AssertionError):
             self.storage.delete('')
 
+    def test_copy_file(self):
+        f = ContentFile('content content')
+        src_name = self.storage.save('initial.file', f)
+        dst_name = self.storage.get_available_name('destination.file')
+        self.storage.copy(src_name, dst_name)
+        self.assertTrue(self.storage.exists(src_name))
+        self.assertTrue(self.storage.exists(dst_name))
+
+    def test_copy_file_no_suspicious_operation(self):
+        f = ContentFile('content content')
+        src_name = self.storage.save('initial.file', f)
+        with self.assertRaises(SuspiciousOperation):
+            self.storage.copy(src_name, '/tmp/passwd')
+
+    def test_copy_raise_on_overwrite(self):
+        f = ContentFile('content content')
+        existing_name = self.storage.save('existing.file', f)
+        src_name = self.storage.save('src.file', f)
+        with self.assertRaises(ValueError):
+            self.storage.copy(src_name, existing_name)
+
+    def test_move_file(self):
+        f = ContentFile('content content')
+        src_name = self.storage.save('initial.file', f)
+        dst_name = self.storage.get_available_name('destination.file')
+        self.storage.move(src_name, dst_name)
+        self.assertFalse(self.storage.exists(src_name))
+        self.assertTrue(self.storage.exists(dst_name))
+
+    def test_move_file_no_suspicious_operation(self):
+        f = ContentFile('content content')
+        src_name = self.storage.save('initial.file', f)
+        with self.assertRaises(SuspiciousOperation):
+            self.storage.move(src_name, '/tmp/passwd')
+
+    def test_move_file_no_directory_traversal(self):
+        f = ContentFile('content content')
+        src_name = self.storage.save('initial.file', f)
+        with self.assertRaises(SuspiciousOperation):
+            self.storage.move(src_name, '../../../../../../../../tmp/passwd')
+
+
 
```

Comment:
Remove one of these blank lines (should only be two total). (`tests/file_storage/tests.py:454:1: E303 too many blank lines (3)`)